apps/remote_control/lib/lexical/remote_control/dispatch.ex

```diff
@@ -63,4 +66,15 @@ defmodule Lexical.RemoteControl.Dispatch do
   defp name do
     {:local, __MODULE__}
   end
+
+  defp register_progress_listener do
+    register_listener(progress_pid(), [project_progress()])
+  end
+
+  defp progress_pid do
+    project = RemoteControl.get_project()
+    master_node_name = RemoteControl.master_node_name(project)
+    progress_name = :rpc.call(master_node_name, Lexical.Server.Project.Progress, :name, [project])
+    :rpc.call(master_node_name, Process, :whereis, [progress_name])
```

Comment:
Worth adding a `Lexical.Server.Project.Progress.whereis(project)` to eliminate some of this back-and-forth?

```suggestion
    :rpc.call(master_node_name, Lexical.Server.Project.Progress, :whereis, [project])
```